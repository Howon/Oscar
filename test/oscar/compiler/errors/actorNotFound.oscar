message testNow()


actor Tester() {
  
  receive = {
    | testNow() => {
        println("Error");
    }
  }
}

def main() => unit = {
  actor<Lost> tester = spawn actor<Lost>();
  message<testNow>() |> tester;
}