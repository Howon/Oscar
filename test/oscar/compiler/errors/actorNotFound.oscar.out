Actor of type Lost not found